internal/label/label.go

```diff
@@ -130,9 +130,26 @@ func (l Label) Equal(other Label) bool {
 		l.Relative == other.Relative
 }
 
-func packageContains(repo, pkg string, label Label) bool {
+// PackageContains tests whether a label is contained within a given repository
+// and package. The label must not be relative.
+func PackageContains(repo, pkg string, label Label) bool {
```

Comment:
I don't like this interface, seems ugly
I think I would have
```
func (l L) Package() {
    return L{
        Repo:     repo,
        Pkg:      pkg,
        Relative: relative,
    }
}
```

and then use
```
mylabel.Package().Equal(label.L{Repo:repo, Pkg: pkg})
```
and ideally the arg to equal would already be a label anyway

Reply:
Changed this to `Contains` method.